### ISSUE TITLE
Group Audio Calls Never Work For Me 

### ISSUE THREAD

**Issue by ralph4q** [qTox/qTox#3248]
##### Group Audio Calls Never Work For Me. People can't hear me, I can't hear anyone, but everyone else in the call hears and talks to eachother fine. Then Tox crashes if I try to hangup or rescan audio devices. I don't get this problem with one-to-one calls, which work fine. I have tried different headsets, they work fine in one to one calls, but not group calls.

OS: Windows 8.1 64
qTox version: v1.3.0
Commit hash:  b69042ec407c30b27ce3e56bb6daf14300facc8e
toxcore:  0.0.0
Qt:  5.5.1
Hardware:  various headsets

Reproducible: ALWAYS 
##### Steps to reproduce
1. Join a Group chat (does not matter whether I am invited to one and accept or create one and invite others)
2. Press call button in group chat
3. Nothing happens -- other people in group hear each other but not me. I don't hear them.
4. Press hang up button
5. TOX crashes
6. TOX also crashes if I go to settings -> audio -> rescan sudio devices during a non-functioning group audio call.

I also have the 32 bit version installed on a windows 10 32 laptop and have the same problem there too.

I'm not a technical person so please forgive me if I haven't provided everything right and let me know in laymans terms if there are other details I can provide that will help this to be fixed.

**Comment by initramfs** [comment]
Can you include the `qtox.log` file? This file is present in `%APPDATA%\tox` if you're on Windows. Additionally, if you can, try loading qTox v1.4.0 and see if the problem still presents. v1.4.0 fixes a lot of group chat related issues.

**Comment by sudden6** [comment]
You are using an old version of qTox, where group audio calls are broken. This should be fixed in v1.4.0

Closing because this is fixed in v1.4.0
